client/src/components/navbar.js

```diff
@@ -1,66 +1,43 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Link, useSearchParams } from 'react-router-dom';
+import png from './icons8-male-user-94.png'
+
+import obook from './open-book.png'
+import cbook from './close-book.png'
+
+import dashboard1 from './icons8-menu-40.png'
 
 const Navbar = () => {
+
+  const [tr, setTr] = useState(false);
+
+  const handleClick = () => {
+    setTr(!tr);
+  }
+
```

Comment:
Can you use good naming conventions.
Like i should be able to undestand the functions or var by its name